package.json

```diff
@@ -100,7 +100,7 @@
     "@github/relative-time-element": "^4.1.2",
     "@lit-labs/react": "^2.0.1",
     "@primer/behaviors": "^1.3.4",
-    "@primer/octicons-react": "^19.3.0",
+    "@primer/octicons-react": "^19.7.0",
```

Comment:
Question: I also have an upgrade PR at dotcom https://github.com/github/github/pull/286206 as well and in the diff, there are so many icons in `node_modules/@primer/react/node_modules/@primer/octicons-react` are deleted. I am a bit confused by this. Does that happen because of npm workspaces that these icons are coming from `node_modules/@primer/octicons-react`?

Reply:
@broccolinisoup I think you're right, now that `@primer/octicons-react` can properly be hoisted we're now seeing all of the ones in the nested `node_modules` being removed.